docs/tctl/schedule/trigger.md

```diff
@@ -0,0 +1,21 @@
+---
+id: trigger
+title: tctl schedule trigger
+sidebar_label: trigger
+description: How to trigger a Schedule Action using tctl
+tags:
+  - tctl
+---
+
+It's a common desire to want a Schedule to start one Workflow run immediately, regardless of its configured Spec.
```

Comment:
```suggestion
It is a common use case to want a Schedule to start one Workflow run immediately, regardless of its configured Spec.
```

Reply:
Or: "Starting a Workflow Run immediately with a Schedule, regardless of its configured Spec, is a common use case."